### ISSUE TITLE
Alignment Accuracy Evaluation Related

### ISSUE THREAD

**Issue by COMPLEXI1** [maiziezhoulab/BenchmarkST#1]
Thank you very much for your work. However, in the current code, I don't seem to find any code related to the accuracy evaluation of the alignment method. I would appreciate it if you could point it out or add it. Thank you again.

**Comment by OliiverHu** [comment]
Hi! Thanks for your interest in our benchmark project. The related codes have been uploaded. I will keep this issue open for a while, so feel free to keep us in the loop if you have any other questions!

Best

**Comment by COMPLEXI1** [comment]
Thank you very much for your reply！

**Comment by COMPLEXI1** [comment]
Hello! I have a small question about the updated code, specifically regarding the Alignment input for the accuracy evaluation function. The example you provided is PASTE, where Alignment can be directly obtained from OT (Optimal Transport). However, for methods like GPSA, there isn't an explicit Alignment output. I checked runGPSA but couldn't find how Alignment is calculated there. Could you please explain how to handle the alignment calculation for methods that do not produce a probability mapping matrix during the alignment process? Thank you!

**Comment by OliiverHu** [comment]
Hi! This is a very good question. For those methods that have a coordinate output (like GPSA). Our makeshift is using the transformed coordinates to calculate the euc dis. The closeness of spots from different slices after transformation could represent the alignment result.

`
coords=df[['aligned_x','aligned_y']].to_numpy()
split_idx = ad1.shape[0]
alignment = sklearn.metrics.pairwise.euclidean_distances(coords[:split_idx], coords[split_idx:])
alignment = np.exp(-alignment)
`

Regards,
